actors/controllers.py

```diff
@@ -92,6 +92,10 @@ def get(self):
                     else:
                         logger.debug("Actor's cron is not activated, but next execution will be incremented")
                         actors_store[actor_id, 'cron_next_ex'] = Actor.set_next_ex(actor, actor_id)
+                elif self.cron_execution_datetime(actor) == 2:
+                    logger.debug(f"Execution time was in the past, now its in the future again")
+                    while self.cron_execution_datetime(actor) ==2: 
```

Comment:
I don't believe this `while` is neccessary, it was just checked and there's no need to run the function again, feel free to flatten this part down.